notebooks/example/anomaly_code.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<p>Can you explain a bit better what is being plotted here? What is 1 Step Prediction vs 5 Step? Why is there no 10 Step prediction? What are the Instantaneous and Likelihood anomalies? Are they the same algorithm with different amounts of training time? Or are they different from each other? Why is "Anomaly Likelihood considered to be the best predictor of Anomaly." ? Are there any performance metrics to back this up?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/HTM-applications/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>